app/os_macos.go

```diff
@@ -15,6 +15,7 @@ import (
 
 	"gioui.org/internal/f32"
 	"gioui.org/io/clipboard"
+	"gioui.org/io/externalDragDrop"
```

Comment:
I would expect this PR referred the `io/transfer` package. See also `widget.Draggable` and the `ExampleDraggable_Layout` example.